Bottom: 809aec71c8c051820776057931e66d073bec580e
Top:    7fc6bb6068bfb3d1225fa8a8cac8f1878b5fdae5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 17:03:16 -0500

Add public key for Andrew Cooper


---
diff --git a/ansible-playbook/public_keys/id_andrew.cooper.pub b/ansible-playbook/public_keys/id_andrew.cooper.pub
new file mode 100644
index 00000000..c2825b4f
--- /dev/null
+++ b/ansible-playbook/public_keys/id_andrew.cooper.pub
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDIUStq5Z2sDke/5qcGe/Tjb03lrqKkXwg1It0GkHY/dk68UJiEjVQTHMicE2D9ML2+QT/ACLmQf7uNDQ4NeNVQUwzyyca8hrVVj0CJYfcQuaUg2hBNLqd9cskiKdXee416bdV8abbDdPs1ZTb/BW45F2HVbde2zq/oqjg122tYPENRSR3hXsEqZcNvBbBGmGy0xGiNt3ejfJkw7lTaEeZlwC74vYMAQIjGZUaoB1rmY6dOYlIQTgU24moyQv5GzA+wI2XFZU0xMHb0Mr3d7pdQrZTs0jmgZkxGeqgoxxbsyusmUVB81kBrPK0uYxU8Iuno0NI/jhy4jz1H/ymndhyF coopera@coopera-xps
